code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/extensions/NotificationCompatBuilderExtensions.kt

```diff
@@ -263,6 +267,7 @@ internal fun NotificationCompat.Builder.addActionButtons(
                     trackerActivityClass,
                     null,
                     eachButton.label,
+                    null,
```

Comment:
If the button action type is DISMISS, should we be passing that value here?

Reply:
you are right, it should be passed here